### DISCUSSION TITLE
How to normalize polar pseudo-coordinates when using GMMConv?

### DISCUSSION THREAD

**Discussion by Icecream-blue-sky** [pyg-team/pytorch_geometric#3892] [Q&A]
I'm using the GMMConv to learn the spatial relation of objects in object detection center. I caculate the polar pseudo-coordinates of each pair of bboxes based on their center coordinate. The operation is like following picture:
![image](https://user-images.githubusercontent.com/26771037/150293600-8197e203-b0c5-41c9-8b44-637c8d7dec28.png)
How should I normalize the pseudo-coordinates before I send them to GMMConv?

**Comment by rusty1s** [comment]
Duplicated discussion: https://github.com/pyg-team/pytorch_geometric/discussions/3903
